lib/findFilesMochaWouldRun.js

```diff
@@ -0,0 +1,85 @@
+// const { join, resolve } = require("path");
+const { resolve } = require("path");
+const minimatch = require("minimatch");
+// TODO: Mocha should be a peerDep
+// TODO: Find out which version of mocha introduced the lookupFiles method.
+const utils = require("mocha/lib/utils");
+
+/* This file is lifted out of `bin/_mocha` [1]. We duplicate mocha's logic to
+ * find the relevant files to build our dependency graph from.
+ * [1]: https://github.com/mochajs/mocha/blob/d401ee198bd5026bc61641ad26eab31e6746713f/bin/_mocha
+ */
```

Comment:
@alexjeffburke This code is as the comment says pulled out of `bin/_mocha`. I can't require that file or other wise get to it.

But notice on the line above where I include the utils file from mocha. It is actually that code that goes and "globs" for the test files.